src/TimeSeries/widget/ui/TimeSeries.css

```diff
@@ -0,0 +1,674 @@
+/* nvd3 version 1.8.4 (https://github.com/novus/nvd3) 2016-07-03 */
```

Comment:
Thanks, yes, webpack rules! This solution was the only way of making widgets. Lets see how I can fix this the good way.